yabt/buildcontext.py

```diff
@@ -396,8 +399,16 @@ def run_in_buildenv(
         docker_run.extend(cmd)
         logger.info('Running command in build env "{}" using command {}',
                     buildenv_target_name, docker_run)
-        # TODO(bergden): accumulate error prints
-        return run(docker_run, check=True, **kwargs)
+        if 'stderr' not in kwargs:
+            kwargs['stderr'] = PIPE
+        if 'stdout' not in kwargs:
+            kwargs['stdout'] = PIPE
+        result = run(docker_run, check=True, **kwargs)
```

Comment:
Currently the fail_notifier prints the stdout and the stderr of the exception. 
But perhaps it would be better if I add a try-except clause to run_in_buildenv which would print the stdout and stderr upon exception if it was set to PIPE?
Something like this: 

        try:
            result = run(docker_run, check=True, **kwargs)
        except Exception as ex:
            if kwargs['stdout'] is PIPE:
                sys.stdout.write(ex.stdout.decode('utf-8'))
            if kwargs['stderr'] is PIPE:
                sys.stderr.write(ex.stderr.decode('utf-8'))
            raise

Or would you recommend a different approach?

Reply:
if `fail_notifier` takes care of printing out stdout/stderr also when an exception is raised here, then it's ok